tests/test_plot_expression_coverage.py

```
@@ -0,0 +1,603 @@
+            strict=False
+        )
+
+        # Should successfully infer keys
+        assert lfc_key is not None
+        assert score_key is not None
+        assert 'mean_lfc' in lfc_key or 'lfc' in lfc_key.lower()
+
+    def test_infer_keys_strict_mode(self):
+        """Test that strict mode raises error when keys can't be inferred."""
+        # AnnData without DE results
+        with pytest.raises(Exception):  # Should raise when keys can't be inferred
```

Ruff (0.14.6)

Do not assert blind exception: `Exception`

(B017)